[ADD][l10n_cl_stock_picking_dte_layout]

gpperez opened the pull request:
Incidence nbr./s.: ODOO-3857 - Error de formato al imprimir Guías de despacho - Almacén San José
- Fix picking_dte_layout report

gpperez committed:
[ADD][l10n_cl_stock_picking_dte_layout]
Incidence nbr./s.: ODOO-3857 - Error de formato al imprimir Guías de despacho - Almacén San José
- Fix picking_dte_layout report

FedericoGregori commented on l10n_cl_stock_picking_dte_layout/.gitignore:
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Este archivo no hace falta a nivel módulo, ya se encuentra a nivel repositorio.

FedericoGregori commented on l10n_cl_stock_picking_dte_layout/README.rst:
```diff
@@ -0,0 +1,92 @@
+===========
```
El nombre de los módulo debe ser en Capital y los signos iguales de la misma longitud que el título

FedericoGregori commented on l10n_cl_stock_picking_dte_layout/README.rst:
```diff
@@ -0,0 +1,92 @@
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-calyx--servicios%2Fcalyx-lightgray.png?logo=github
```
Corregir el badge para que indique el repositorio correspondiente

FedericoGregori replied:
calyx -> l10n--cl

FedericoGregori commented on l10n_cl_stock_picking_dte_layout/README.rst:
```diff
@@ -0,0 +1,92 @@
+
+.. !!! Description must be max 2-3 paragraphs, and is required.
+
+This module extends the functionality of ... to support ... and to allow you to ...
```
Corregir descripción del módulo

FedericoGregori commented on l10n_cl_stock_picking_dte_layout/README.rst:
```diff
@@ -0,0 +1,92 @@
+CALYX SERVICIOS S.A. is part of the PGK Consultores economic group, member of an important global network, a world organization.
+The PGK Consultores group is one of the 20 largest consultant-studios in Argentina with nearly 300 professionals.
+
+This module is part of the `Calyx <https://github.com/calyx-servicios/calyx.git>`_ project on GitHub.
```
Corregir link y texto a mostrar

FedericoGregori commented on l10n_cl_stock_picking_dte_layout/__init__.py:
```diff
@@ -0,0 +1 @@
+#from . import models
```
Borrar comentarios sin información relevante

FedericoGregori commented on l10n_cl_stock_picking_dte_layout/__manifest__.py:
```diff
@@ -0,0 +1,24 @@
+    "development_status": "Production/Stable",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
```
Borrar keys no usadas

FedericoGregori commented on l10n_cl_stock_picking_dte_layout/views/layout.xml:
```diff
@@ -0,0 +1,332 @@
+        </xpath>
+    </template>
+
+    <!--<template id="dte_layout" inherit_id="l10n_cl_stock_picking.dte_layout">
```
Borrar comentarios no relevantes. Si se desea dejar comentado una parte del código, explicar por qué

gpperez committed:
[ADD][l10n_cl_stock_picking_dte_layout]
Incidence nbr./s.: ODOO-3857 - Error de formato al imprimir Guías de despacho - Almacén San José

Fix picking_dte_layout report
